app/queries/servers/post.ts

```diff
@@ -142,7 +142,7 @@ export const queryPostsBetween = (database: Database, earliest: number, latest:
         andClauses.push(Q.where('user_id', userId));
     }
 
-    if (rootId) {
+    if (rootId !== undefined) {
```

Comment:
this is allowing to query for null which will give an error, you should probably check for `rootId != null` or `typeof rootId === 'string')` I prefer the first option.